net/node/transactionPool.go

```diff
@@ -135,9 +135,9 @@ func (this *TXNPool) apendToUTXOPool(txn *transaction.Transaction) (bool, *trans
 		return false, nil
 	}
 	for k, _ := range reference {
-		txn := this.getInputUTXOList(k)
-		if txn != nil {
-			return false, txn
+		tx := this.getInputUTXOList(k)
```

Comment:
Good catch,  suggest rename the txn to t or transaction as the tx already used for mark "network transmit"